MetaMask/metamask-extension
# Replacing deprecated constants and components

## Explanation

This PR is a part of the following issues
- #18714 
- #19526 
- #18896 

Made changes to the `UI` folder, the folders involved are
1. `ui/components/ui/deprecated-test-networks`
2. `ui/components/ui/nft-info`

## Screenshots/Screencaps

### `nft-info`
### Before
<img width="741" alt="Screenshot 2023-07-28 at 11 43 41 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/efd5513c-ff2f-408e-9017-72fa4b0daa1c">

### After
<img width="1228" alt="Screenshot 2023-07-28 at 11 44 05 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/1ab5e27d-0d28-4381-a30c-c2fe8d3b9606">

## Manual Testing Steps

- Pull this branch
- Run `yarn storybook`
- Check listed stories

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
